package.json

```diff
@@ -7,7 +7,7 @@
     "start": "node build/index.js",
     "test": "jest --forceExit --verbose",
     "coverage": "jest --forceExit --coverage && cat ./coverage/lcov.info | coveralls",
-    "dev": "cross-env DEBUG=dev && nodemon --exec babel-node server/index.js",
+    "dev": "set DEBUG=dev && nodemon --exec babel-node server/index.js",
```

Comment:
Why did you change from cross-env?

Reply:
It was failing in my end, saying cross-env wasn't recognized

Reply:
You should run npm install before using the cross-env. It's a package. Changing it to set will affect those of us using non-windows machines.

Reply:
I actually ran npm install before working, that's why my cors could work. I don't why it didn't work. I would try it again

Reply:
All right bro. Pls do